tests/dummy/app/helpers/eq.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export function eq(params) {
```

Comment:
Please go ahead and install `ember-truth-helpers` (which provides this helper and many other useful ones too)